### PR TITLE
Fix position of 6DoF controllers and beam position

### PR DESCRIPTION
**Pull request by bluemarvin**

### CODE REVIEW

File: app/src/main/cpp/ControllerContainer.cpp [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include <assert.h>
+#include <cassert>
```
Comment (daoshengmu) [inline app/src/main/cpp/ControllerContainer.cpp]: do we need this change?

Comment (daoshengmu) [inline app/src/main/cpp/ControllerContainer.cpp]: Do we need this change?

Comment (bluemarvin) [reply app/src/main/cpp/ControllerContainer.cpp]: It's not needed at all, I removed the `#include` altogether.

**Comment by philip-lamb** [comment]
Fixes https://github.com/MozillaReality/FirefoxReality-SecureBugs/issues/4

**Pull request merged**